Helpers/get_results_from_criteria_and_data.py

```diff
@@ -0,0 +1,78 @@
+from datetime import date, timedelta
+from typing import List
+
+from graphql_objects import DatesCheck
+
+
+class GetResultsFromSearchCriteriaAndData():
+
+    @staticmethod
+    def get_data_check_results(validated_search_criteria, id_houses_dict: dict) -> List[DatesCheck]:
+        results_list: List[DatesCheck] = []
+        checkin_desired = validated_search_criteria.desired_checkin
+        checkout_desired = validated_search_criteria.desired_checkout
+        for house in id_houses_dict.values():
+            data_check = DatesCheck(listing_id=house.listing_id,
+                                    is_available=True,
+                                    checkin_date=checkin_desired,
+                                    checkout_date=checkout_desired)
```

Comment:
To be stylistically consistent, we prefer:
```
data_check = DatesCheck(
    listing_id=house.listing_id,
    is_available=True,
    checkin_date=checkin_desired,
    checkout_date=checkout_desired
)
```

Reply:
Also, be consistent in your naming. Why is it `desired_checkin` in your input but `checkin_desired` here?